src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
```

Comment:
**The task is not done. Do not use raw type.**
![image](https://user-images.githubusercontent.com/20326858/209575083-8706007d-0124-4156-bd70-2b153d615c38.png)

Reply:
not fixed